ui/app/src/services/users.ts

```diff
@@ -217,7 +217,7 @@ export function hasPermissions(site: string, ...permissions: string[]): Observab
 }
 
 export function hasPermission(site: string, permission: string): Observable<boolean> {
-  return hasPermissions(site, permission).pipe(map((response) => response?.permission));
```

Comment:
this service and `hasGlobalPermission` are not in use, should we remove then or leave them (as they are possible usages in the future)?

Reply:
Leave